.github/ISSUE_TEMPLATE/qsl_bug_report.yml

```diff
@@ -0,0 +1,43 @@
+name: QSL-specific Bug Report
+description: Create a bug report to help us fix issues with Quilt Standard Libraries.
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking your time to report this bug! If you need real-time help, join us on [Discord](https://discord.quiltmc.org).
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: "Describe the bug as clearly as possible. The more detail you provide means the sooner the bug can be fixed! This includes: steps to reproduce or situation you encountered the bug in, recent changes you made to your mods list, or any additional context that could be helpful."
+    validations:
+      required: true
+
+  - type: textarea
+    id: where-in-qsl
+    attributes:
+      label: How did QSL break?
+      description: "What part of the game broke? Which QSL modules were in the stack trace? Is it possible to provide a GitHub link to the section of QSL that is broken?"
+    validations:
+      required: true
+
+  - type: dropdown
+    id: environment
+    attributes:
+      label: Which environment did you encounter this bug in?
+      options:
+        - Client
+        - Server
+        - Both
+    validations:
+      required: true
+
+  - type: input
+    id: logs
+    attributes:
+      label: Logs
+      description: |
+        If applicable, please provide your crash report or latest.log
```

Comment:
quick question: are crash reports sometimes needed for fixing QSL problems?

Reply:
Yes, if QSL caused the crash

Reply:
oh ok